assets/js/modules/analytics-4/utils/data-mock.js

```diff
@@ -25,6 +25,7 @@ import invariant from 'invariant';
 import castArray from 'lodash/castArray';
 import { zip, from, Observable } from 'rxjs';
 import { map, reduce, take } from 'rxjs/operators';
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
```suggestion
import cloneDeep from 'lodash/cloneDeep';
```

Given #6492 is already in the backlog, let's not add more usages of `cloneDeep`. We can convert these calls to `structuredClone`, right?

Reply:
Unfortunately, not yet as it turns out. I did try `structuredClone` initially but got a linting error, so I figured I'd stick with `cloneDeep` and leave the details to be sorted out in #6492. However, I've now dug into it a bit further and found that in `structuredClone` is only available in Node from version 17. With that in mind I've left a note on #6492 and sent it back your way for consideration as we should probably factor this into the IB.

![image](https://user-images.githubusercontent.com/18395600/216591708-cc50c5cf-ad82-4eb4-8cf8-7a4716b081c1.png)

Reply:
Ah, nice find! 👍🏻